DataFormats/PatCandidates/interface/Photon.h

```diff
@@ -169,6 +170,11 @@ namespace pat {
 	    //<< key << " was not stored for this particle.";
           }
       }
+
+      float puppiChargedHadronIso() const {return puppiChargedHadronIso_; };
+      float puppiNeutralHadronIso() const {return puppiNeutralHadronIso_; };
+      float puppiPhotonIso() const {return puppiPhotonIso_; };
+
```

Comment:
@nhanvtran @violatingcp 
please check this
I'd like to confirm that photon puppi iso will never have to be computed wrt puppiNoLeptons by design.
If we will need it in some analyses, then corresponding variables will need to be added to the pat::Photon